atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -201,6 +190,59 @@ def add_as_an_alias(new, old, is_old_inherited):
             self.log.debug("applied only some aliases, following old labels were not found: %s",
                            ", ".join(not_applied))
 
+        def check_if_all_same(labels_to_check, source):
+            if labels_to_check:
+                label_value = source[next(iter(labels_to_check))]
```

Comment:
yeah both are viable options, I just wanted to avoid to create additional list or set, and fail as soon I've find any non equal